building-and-research/metrics/bertscore.md

```diff
@@ -14,11 +14,11 @@ BERTScore was invented as an improvement on [n-gram](/n-gram)-based metrics like
 >
 > [...] First, such methods often fail to robustly match paraphrases.
 >
-> [...] Second, n-gram models fail to capture distant dependencies and penalize semantically-critical ordering changes.
+> [...] Second, n-gram models fail to capture distant dependencies and penalise semantically-critical ordering changes.
```

Comment:
These are quotations taken directly from the paper. In this and several other cases throughout this PR, there are instances of literal quotations. I'm not sure about changing their spellings.

Reply:
Right, we should not.